src/components/Sidebar.js

```diff
@@ -116,10 +120,18 @@ const StatusText = styled.div`
 
 const SponsorLogo = styled.img`
   display: block;
-  margin: 1em 0 0 60px;
+  margin: 1em 0 0 50px;
```

Comment:
Random request, but since you're already working in sidebar would you mind fixing the margin on the "Logout" button as well? It seems like it's not aligned